lib/config/dpl-cms/dplCmsConfig.ts

```diff
@@ -77,6 +80,9 @@ export const getDplCmsPublicConfig = async () => {
       logoutUrls: {
         adgangsplatformen: null,
       },
+      libraryInfo: {
+        name: "",
```

Comment:
I think we should make the default libraryInfo.name a nullable string (like the loginUrls/logoutUrls).
By doing that we are sure that when we are resolving the value we know that we did our best to retrieve the value, we did not succeed and mark it as `null`.
If we are using a string as our fallback value we do not know if we actually succeeded in retrieving the value, but for some weird reson it was not set in the dpl-cms.
Does it make sense?

Reply:
yesssssir :)